lib/v2/screens/profile_screens/profile/components/citizenship_upgrade_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/screens/profile_screens/profile/interactor/viewmodels/profile_state.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+
+class CitizenshipUpgradeButton extends StatelessWidget {
+  final CitizenshipUpgradeStatus citizenshipUpgradeStatus;
+
+  const CitizenshipUpgradeButton({
+    Key? key,
+    required this.citizenshipUpgradeStatus,
+  }) : super(key: key);
```

Comment:
It is not necessary to make it named parameter since it is the only parameter it receives.

`  const CitizenshipUpgradeButton(this.citizenshipUpgradeStatus, {Key? key}) : super(key: key);`

Reply:
always forget that

Reply:
fix